.github/workflows/ci.yml

```diff
@@ -458,22 +419,13 @@ jobs:
       - name: Limit ray memory
         run: echo "MODIN_MEMORY=1000000000" >> $GITHUB_ENV
```

Comment:
Can we remove it at all since we will use `./.github/actions/use-ray-cluster`?

Reply:
They're executed under different conditions, this is for non-Windows while `use-ray-cluster` is for Windows